### PR TITLE
chore(frontend): Use Tailwind's base

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Now that we're off Ant (#24256), we can use Tailwind's preflight styles, which include a bunch of sane normalized defaults. Allows us to remove some of our homegrown or borrowed from Ant normalization.
Reducing the UI snapshot difference threshold to catch anything that might be off.

### CODE REVIEW

**Review by daibhin — DISMISSED** [review]

File: frontend/__snapshots__/filters-propertyfilters--comparing-property-filters--light.png
Comment (daibhin) [inline frontend/__snapshots__/filters-propertyfilters--comparing-property-filters--light.png:1]: Looks like we lost some margin here

File: frontend/__snapshots__/components-activitylog--data-management-activity--dark.png
Comment (daibhin) [inline frontend/__snapshots__/components-activitylog--data-management-activity--dark.png:1]: No strong opinion, just checking that we're comfortable with headings changing size like this

File: frontend/__snapshots__/components-playerinspector-itemevent--error-event--dark.png
Comment (daibhin) [inline frontend/__snapshots__/components-playerinspector-itemevent--error-event--dark.png:1]: Button on the right hand side seems to move but the text stays in the same place

File: frontend/__snapshots__/lemon-ui-lemon-calendar-lemon-calendar--hour--light.png
Comment (daibhin) [inline frontend/__snapshots__/lemon-ui-lemon-calendar-lemon-calendar--hour--light.png:1]: Not sure which looks better but certainly different

File: frontend/__snapshots__/scenes-other-invitesignup--logged-in-wrong-user--light.png
Comment (daibhin) [inline frontend/__snapshots__/scenes-other-invitesignup--logged-in-wrong-user--light.png:1]: Any idea why the padding has changed on all of these?

File: frontend/__snapshots__/scenes-other-signup--self-hosted-sso--light.png
Comment (daibhin) [inline frontend/__snapshots__/scenes-other-signup--self-hosted-sso--light.png:1]: Form on the right has different padding

**Review by daibhin — APPROVED** [review]
Difficult to review everything so I just did spot checks. Certainly some differences but nothing looked broken or missing. Left a few comments where I saw things but happy to approve & fix forward (if at all) because this could be a rebase nightmare otherwise

File: frontend/__snapshots__/components-playerinspector-itemevent--error-event--dark.png
Comment (Twixes) [reply frontend/__snapshots__/components-playerinspector-itemevent--error-event--dark.png:1]: Verified that this is just an artifact of the screenshot, some pixel rounding

**Pull request merged**